Examples/Cxx/Circuits/DP_Slack_PiLine_VSI_with_PF_Init.cpp

```diff
@@ -97,7 +97,7 @@ int main(int argc, char* argv[]) {
 	auto n2DP = SimNode<Complex>::make("n2", PhaseType::Single);
 
 	auto extnetDP = DP::Ph1::NetworkInjection::make("Slack", Logger::Level::debug);
-	extnetDP->setParameters(Complex(scenario.systemNominalVoltage,0));
+	extnetDP->setParameters(Complex(scenario.systemNominalVoltage,0), 0.0);
```

Comment:
Not required, just set for clarification I assume?

Reply:
Yes, but I'll remove it anyway.